sos/report/plugins/ceph_podman.py

```diff
@@ -0,0 +1,110 @@
+# Copyright (C) 2020 Meiyan Zheng <mzheng@redhat.com>
+#
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.report.plugins import Plugin, RedHatPlugin
+import re
+
+
+class CephPodman(Plugin, RedHatPlugin):
+    short_desc = 'Containerized Ceph distributed storage'
+    plugin_name = 'ceph_podman'
+    profiles = ('storage', 'container')
+    packages = ('podman',)
```

Comment:
This would cause non-ceph nodes that have `podman` installed to always run this plugin.